CHANGELOG.md

```diff
@@ -6,6 +6,20 @@
 
 #### Improvements
 - Added support server side string size limitations.
+- Added support for environment variable `SNOWPARK_SPAN_RECORD` that user can decide whether record open telemetry span. Recording is enabled when `SNOWPARK_SPAN_RECORD` is not set or set to `enable`, it is disabled when `SNOWPARK_SPAN_RECORD` set to `disable`. The following APIs create spans:
```

Comment:
Better align with python connector connections toml and other SDKs for this name (SNOWPARK_SPAN_RECORD). We don't want users to config Snowflfake's SDKs one by one. A single one for all of them is better.

Reply:
`SNOWFLAKE_...` may be better. See here https://docs.snowflake.com/en/developer-guide/python-connector/python-connector-connect